packages/services/api/src/modules/organization/module.graphql.ts

```diff
@@ -57,6 +58,19 @@ export default gql`
     inputErrors: CreateOrganizationInputErrors!
   }
 
+  type LeaveOrganizationResult {
+    ok: LeaveOrganizationOk
+    error: LeaveOrganizationError
+  }
+
+  type LeaveOrganizationOk {
+    id: ID!
```

Comment:
should be `organizationId`? calling it `id` might imply that this is an entity? :D